app/src/components/ProtvistaMSA.jsx

```diff
@@ -174,68 +159,46 @@ const ProtvistaMSAWrapper = () => {
             onChange={(evt) => setSampleSizeConservation(evt.target.value)}
           />
         </label>
-        <label>
-          show left coordinates:
-          <input
-            type="checkbox"
-            checked={showLeftCoordinate}
-            onChange={() => setShowLeftCoordinate(!showLeftCoordinate)}
-          />
-        </label>
-        <label>
-          show right coordinates:
-          <input
-            type="checkbox"
-            checked={showRightCoordinate}
-            onChange={() => setShowRightCoordinate(!showRightCoordinate)}
-          />
-        </label>
-        {(showLeftCoordinate || showRightCoordinate) && (
-          <>
-            <label>
-              offset coordinates by sequence start:
-              <input
-                type="checkbox"
-                checked={offsetSeqStart}
-                onChange={() => setOffsetSeqStart(!offsetSeqStart)}
-              />
-            </label>
-            <label>
-              exclude gaps from coordinate count:
-              <input
-                type="checkbox"
-                checked={excludeGaps}
-                onChange={() => setExcludeGaps(!excludeGaps)}
-              />
-            </label>
-          </>
-        )}
       </div>
       <protvista-manager
         attributes="length displaystart displayend highlight"
         displaystart="1"
-        displayend="50"
+        displayend="100"
         id="example"
       >
         <div style={{ display: "flex", width: "100%" }}>
           <div
             style={{
-              width: labelWidth + (showLeftCoordinate && coordinateWidth),
+              width: labelWidth,
               flexShrink: 0,
             }}
           />
-          <protvista-navigation
-            length={sequence.length + 1}
-            displaystart="1"
-            displayend="50"
-          />
+          <protvista-navigation length={sequence.length} />
+        </div>
+        <div style={{ display: "flex", width: "100%" }}>
           <div
             style={{
-              width: showRightCoordinate && coordinateWidth,
+              width: labelWidth,
               flexShrink: 0,
+              height: "20px",
+              textAlign: "left",
+              overflow: "hidden",
+              fontWeight: "bold",
+              fontSize: "14px",
+              color: "rgb(0, 99, 154)",
+              textTransform: "uppercase",
             }}
+          >
+            Contacts
+          </div>
+          <protvista-links
+            id="links-track"
+            length={sequence.length}
+            height={20}
+            use-ctrl-to-zoom
```

Comment:
Is this needed?

Reply:
do you mean the `use-ctrl-to-zoom` attribute?
If so, it's not needed, it is a matter of preference. 
Zooming in the track is an inherited feature from `protvista-zoomable`, which by default uses scrolling gestures to zoom in/out. We had several complains of users saying that behaviour was disrupting because they wanted to scroll and not to zoom. so we add the `use-ctrl-to-zoom` attribute, to say that this track would use scrolling gestures for zooming only if the `[ctrl]` key is pressed.

Reply:
> Have you updated the `react-msa-viewer`, the errors you describe sound to me as if is using a previous version of it.

You're quite right. I had deleted all of the `node_modules` directories and ran `yarn` but I guess this wasn't enough so I recloned the `nightingale` and removed the `react-msa-viewer` repo.